app/views/users/_user_card.html.erb

```diff
@@ -0,0 +1,45 @@
+<style>
+  .all {
+    display: flex;
+    height: 100px;
+    justify-content: space-between;
+    margin: 10px 0;
+  }
+
+  .all > div {
+    border: 1px solid black;
+  }
+
+  .imgDiv {
+    width: 20%;
+  }
+
+  img {
+    width: 100%;
+  }
+
+  .userDiv {
+    width: 70%;
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .comments {
+    align-self: flex-end;
+  }
+</style>
```

Comment:
[OPTIONAL]  Inline styling is one of CSS bad practices. When you add inline styles, it makes it difficult revising the design of your site and makes your code harder to manage.
I would recommend moving the style to the assets folder.